tools/autograd/gen_autograd_functions.py

```diff
@@ -79,6 +83,17 @@
   ${body}
   return grad_inputs;
 }
+#ifdef COMPILED_AUTOGRAD
```

Comment:
Should this be prefixed with `TORCH_` ? Or at least `USE_` to match our other macros.
Also this needs to be documented in setup.py

Reply:
This intentionally isn't exposed in setup.py.  It is defined in function.h based on `_WIN32`.

Reply:
> This intentionally isn't exposed in setup.py.

Should it? How painful is it going to be to debug build issues related to that if you can only repro with the windows machines on CI?

Reply:
I don't think the symbol limit is an issue anywhere other than windows, so you need a windows machine to debug it.